### DISCUSSION TITLE
Create image with different dimensions but same pixel type

### DISCUSSION THREAD

**Discussion by jurgen-kluft** [SixLabors/ImageSharp#2391] [Q&A]
Image img = Image.Load("test.png");

// how do I create an image with a certain size but the same pixel type of 'img'?
Image another = new Image<?>(anotherWidth, anotherHeight);

**Comment by JimBobSquarePants** [comment]
The pixel format isn't exposed in the non-generic type. You would have to use the [`Image.PixelType`](https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.Image.html#SixLabors_ImageSharp_Image_PixelType) property to determine the bit depth and alpha component representation of the pixel format and make a best guess.

Otherwise enforce a pixel format by avoiding the non-generic type on load. 

 ```c#
 using Image img = Image.Load<Rgba32>("test.png");
```

**Reply by jurgen-kluft** [reply]
Ok
